charts/argo-cd/values.yaml

```diff
@@ -784,9 +784,11 @@ redis-ha:
     enabled: true
     # -- Exporter image
     # @default -- `nil` (follows subchart default)
+    ## Note that default value (nil) causes error when using kustomize, so please make sure to override it as a workaround.
```

Comment:
Is it better to note on the top of [redis-ha](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L776-L778) at once...? 🤔 (but maybe users would miss it ?)